extensions/mapstore/src/test/java/com/hazelcast/mapstore/AllTypesGenericMapStoreTest.java

```diff
@@ -61,65 +66,82 @@ public class AllTypesGenericMapStoreTest extends JdbcSqlTestSupport {
     public String value;
 
     @Parameterized.Parameter(2)
-    public BiConsumer<GenericRecord, String> verify;
+    public Object singleColumn;
 
     @Parameterized.Parameter(3)
-    public BiConsumer<GenericRecordBuilder, String> setField;
+    public BiConsumer<GenericRecord, String> verify;
+
+    @Parameterized.Parameter(4)
+    public Consumer<Tuple3<GenericRecordBuilder, String, Object>> setField;
+
+    @Parameterized.Parameter(5)
+    public Consumer<Object> verifySingleColumn;
+
 
     private String tableName;
-    private GenericMapStore<Integer> mapStore;
+    private String tableNameSingleColumn;
```

Comment:
at this point I'd extract single-column tests to separate class, then assertions also will be in test method instead of consumers